en/08_Changelogs/6.0.0.md

```diff
@@ -107,6 +109,17 @@ The code in `silverstripe/externallinks`, `silverstripe/security-report`, and `s
 
 CMS 6 compatible versions of `silverstripe/blog`, `silverstripe/subsites`, and `silverstripe/crontask` have been released with CMS 6.0.0, though they are not commercially supported.
 
+### New supported modules
+
+Some new modules have been created and are now covered by our [supported modules policy](/project_governance/supported_modules/). These modules will be supported for the full lifetime of Silverstripe CMS 6.
+
+> [!WARNING]
+> To continue using TinyMCE as an HTML editor in your existing project, you must add `silverstripe/htmleditor-tinymce` as an explicit dependency.
```

Comment:
Intentionally putting this warning both here AND below so people have plenty of opportunity to see it.